examples/cortex-m-hello-world/src/main.rs

```diff
@@ -1,29 +1,32 @@
 #![no_std]
 #![no_main]
-#![allow(non_upper_case_globals)]
 
 extern crate panic_semihosting;
 
 use cortex_m::peripheral::syst::SystClkSource;
 use cortex_m_rt::{entry, exception};
 
-use aerugo::{AERUGO, InitApi, MessageQueueStorage, TaskletConfig, TaskletStorage};
+use aerugo::{InitApi, MessageQueueStorage, TaskletConfig, TaskletStorage, AERUGO};
 
 #[allow(dead_code)]
 struct TaskAData {
     a: u8,
     b: u32,
 }
 
+fn task_a(_: u8) {}
+
 #[allow(dead_code)]
 struct TaskBData {
     a: u16,
     b: u16,
 }
 
-static tasklet_a: TaskletStorage<u8, TaskAData> = TaskletStorage::new();
-static tasklet_b: TaskletStorage<u8, TaskBData> = TaskletStorage::new();
-static queue_x: MessageQueueStorage<u8, 16> = MessageQueueStorage::new();
+fn task_b(_: u8) {}
+
+static TASK_A: TaskletStorage<u8, TaskAData> = TaskletStorage::new();
```

Comment:
I don't like the fact that storage has exactly the same name as function - add `_STORAGE` suffix?

Reply:
done